var/spack/repos/builtin/packages/py-mypy/package.py

```diff
@@ -20,10 +23,8 @@ class PyMypy(PythonPackage):
     version('0.740', sha256='48c8bc99380575deb39f5d3400ebb6a8a1cb5cc669bbba4d3bb30f904e0a0e7d')
     version('0.670', sha256='e80fd6af34614a0e898a57f14296d0dacb584648f0339c2e000ddbf0f4cc2f8d')
 
-    variant('python2', default=False, description='Enable checking of python 2 type annotations')
```

Comment:
This variant was never used anywhere. The pip package does have a python2 variant which installs typed-ast (regardless of Python version). Not sure if it's worth keeping the variant and adding that.